### ISSUE TITLE
Add Pretendo Support 

### ISSUE THREAD

**Issue by TheBrockReal** [MCMi460/3DS-RPC#29]
The Nintendo Network is officially dead, it's time to add Pretendo Support

**Comment by Preloading** [comment]
Hi there!

1. Duplicate of #10 and #23
2. It's basically done, we are just waiting on MCMi460 to review my pr (#28), and for pretendo to fix a bug with there systems
3. The nintendo network part STILL WORKS, they haven't nuked the friends server yet.

**Comment by spotlightishere** [comment]
This is currently deployed in production - we'll make a proper announcement after more testing to ensure everything is working smoothly. Thank you so much for the issue!
